apps/builder/app/builder/features/style-panel/sections/transitions/transition-utils.ts

```diff
@@ -86,6 +89,30 @@ export const findTimingFunctionFromValue = (
   );
 };
 
+export const getAnimatablePropertiesOnInstance = (
```

Comment:
@TrySound due to your recent changes, does this need any change?

Reply:
I'll do it later myself